src/components/ViewerSideNav/ViewerSideNav.tsx

```diff
@@ -56,45 +80,51 @@ const SideNavbar: React.FC<SidenavProps> = ({ items }) => {
         <DrawerOverlay onClick={closeSideNav} />
       </CSSTransition>
       <HamburgerButton active={expanded} onClick={() => setExpanded(!expanded)} />
-      <SidebarNav expanded={expanded}>
+      <SidebarNav expanded={expanded} isStudio={isStudio}>
         <LogoLink to="/" onClick={closeSideNav} tabIndex={expanded ? 0 : -1}>
           <Logo />
+          {isStudio && <StudioText>studio</StudioText>}
         </LogoLink>
         <SidebarNavList>
-          {items.map((item) => (
+          {navItems.map((item) => (
             <NavItem
               key={item.name}
               to={item.to}
               expanded={expanded}
               subitems={item.subitems}
               itemName={item.name}
               onClick={closeSideNav}
+              isStudio={isStudio}
             >
               <Icon name={item.icon} />
-              <span>{item.name}</span>
+              <span>{expanded && item.expandedName ? item.expandedName : item.name}</span>
```

Comment:
This doesn't work very well because you can see the change when collapsing. Do we need to change those at all? I think this should work fine if we displayed `expandedName` always?

Reply:
@mikkio-j I think you misunderstood what I said here. I didn't mean to remove `expandedName` altogether, this clearly looks bad:
<img width="101" alt="Screenshot 2021-03-23 at 17 35 27" src="https://user-images.githubusercontent.com/12646744/112183083-33168500-8bfe-11eb-80a7-4673d91f9672.png">
What I meant was that we can always display the `expandedName` in that `span` the comment was targeting, we should still pass the shorter name to the `NavItem`